3460/0/triangle.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+# Triangle Project Code.
+
+# Triangle analyzes the lengths of the sides of a triangle
+# (represented by a, b and c) and returns the type of triangle.
+#
+# It returns:
+#   :equilateral  if all sides are equal
+#   :isosceles    if exactly 2 sides are equal
+#   :scalene      if no sides are equal
+#
+# The tests for this method can be found in
+#   about_triangle_project.rb
+# and
+#   about_triangle_project_2.rb
+# rubocop:disable CyclomaticComplexity
+# rubocop:disable PerceivedComplexity
+#
+
+def triangle(side_a, side_b, side_c)
+  variable_one = side_a + side_b
```

Comment:
N-naming
Hint: u can create additional methods

Reply:
if I understood you correctly, then I fixed